deploy/sample-backup-configs/sample_config_runtime.yaml

```diff
@@ -1,17 +1,13 @@
 # RUNTIME configuration for the FireSim Simulation Manager
 # See https://docs.fires.im/en/stable/Advanced-Usage/Manager/Manager-Configuration-Files.html for documentation of all of these params.
 
-run_farm:
-    run_farm_tag: mainrunfarm
-    always_expand_runfarm: yes
-    f1_16xlarges: 1
-    m4_16xlarges: 0
-    f1_4xlarges: 0
-    f1_2xlarges: 0
-    launch_instances_timeout_minutes: 60
-    run_instance_market: ondemand
-    spot_interruption_behavior: terminate
-    spot_max_price: ondemand
+run_farm_config:
+  # managerinit replace start
```

Comment:
The `managerinit replace start/end` is used to replace this section in `managerinit`. Naming could potentially change here:
- `run_farm_config`? `run_farm`? `run_farm_recipe`?
- `defaults`? `default_recipe`?
- `override_args`? `overrides`?

Reply:
Make sure you capture that in the docs.

Reply:
my 2c: 

> * `run_farm_config`? `run_farm`? `run_farm_recipe`?
`run_farm`
> * `defaults`? `default_recipe`? `recipe`?
I personally like `base` or `base_recipe`. 
> * `override_args`? `overrides`?
`overrides`

Reply:
I also like `run_farm` and `base_recipe`, but for `override_args`, I would prefer something more verbose, e.g. `recipe_arg_overrides`.

Also, I would add a comment that emphasizes that the expected structure of the contents of `overrides` is identical to that of the recipe you're overriding parts of.

Reply:
I tend to agree that "args" should also be present since I want to emphasize that you can only override the "args" section of the run farm recipe. I have little preference on the "recipe_*" or just "arg_overrides"

Reply:
Also updated the comment on `run_farm_type`. This will be made clear in the follow on documentation.

Reply:
Settled on `run_farm, base_recipe, recipe_arg_overrides`. Closing